pkg/scanfetcher/nexpose.go

```diff
@@ -129,6 +130,11 @@ func (n *NexposeClient) makePagedNexposeScanRequest(page int) (nexposeScanRespon
 		return nexposeScanResponse{}, err
 	}
 
+	if res.StatusCode != http.StatusOK {
+		return nexposeScanResponse{}, fmt.Errorf("unexpected response from nexpose scans api: %d %s",
```

Comment:
Question: Would is be useful to have a custom error for this?

Reply:
We're not checking the error type returned in the handler, so I'm not sure what value it would provide.